### DISCUSSION TITLE
Increment not work as I expected

### DISCUSSION THREAD

**Discussion by tofarfar** [Hexagon/croner#223] [Q&A]
25 15 2/3 * *

The above expression run at 2023/11/03、2023/11/06 etc, but my expectation is 2023/11/02、2023/11/05 etc.
Croner seems to ignore 2 in 2/3, is this a feature or bug?

Thanks!

**Comment by Hexagon** [comment]
Interesting, I'd consider that a bug, transfering to issues 👍
